vault/assets/dashboards/vault_overview.json

```
@@ -1,594 +1,920 @@
+                              "formula": "query4"
+                          }
+                      ],
+                      "on_right_yaxis": false,
+                      "queries": [
+                          {
+                              "data_source": "metrics",
+                              "name": "query4",
+                              "query": "sum:vault.vault.core.handle.request.count{$vault_cluster,$api_url}.as_count()"
+                          }
+                      ],
+                      "response_format": "timeseries",
+                      "style": {
+                          "line_type": "solid",
+                          "line_width": "normal",
+                          "palette": "dog_classic"
+                      }
+                  }
+              ],
+              "show_legend": true,
+              "title": "Number of requests over $vault_cluster,$api_cluster",
+              "title_align": "left",
+              "title_size": "16",
+              "type": "timeseries",
+              "yaxis": {
+                  "include_zero": true,
+                  "label": "",
+                  "max": "auto",
+                  "min": "auto",
+                  "scale": "linear"
               }
-            }
-          ],
-          "custom_links": [],
-          "title": "Number of leases",
-          "title_size": "16",
-          "title_align": "center",
-          "show_legend": false,
-          "legend_size": "0"
-        },
-        "layout": {
-          "x": 88,
-          "y": 58,
-          "width": 63,
-          "height": 18
-        }
+          },
+          "id": 3036011336694786,
+          "layout": {
+              "height": 15,
+              "width": 47,
+              "x": 73,
+              "y": 22
+          }
       },
       {
-        "id": 14,
-        "definition": {
-          "type": "note",
-          "content": "Leases",
-          "background_color": "gray",
-          "font_size": "24",
-          "text_align": "center",
-          "show_tick": true,
-          "tick_pos": "50%",
-          "tick_edge": "bottom"
-        },
-        "layout": {
-          "x": 88,
-          "y": 50,
-          "width": 63,
-          "height": 7
-        }
+          "definition": {
+              "background_color": "gray",
+              "content": "## Vault\n\nThis dashboard provides a high-level overview of your Vault clusters so you can monitor its performance and cluster health.\n\n- [Official Vault integration docs](https://docs.datadoghq.com/integrations/vault/)\n- [Monitoring HashiCorp Vault with Datadog](https://www.datadoghq.com/blog/monitor-hashicorp-vault-with-datadog/)",
+              "font_size": "14",
+              "has_padding": true,
+              "show_tick": true,
+              "text_align": "left",
+              "tick_edge": "right",
+              "tick_pos": "50%",
+              "type": "note",
+              "vertical_align": "center"
+          },
```

```suggestion
          "definition": {
              "background_color": "neutral",
              "content": "## Vault\n\nThis dashboard provides a high-level overview of your Vault clusters so you can monitor its performance and cluster health.\n\n- [Official Vault integration docs](https://docs.datadoghq.com/integrations/vault/)\n- [Monitoring HashiCorp Vault with Datadog](https://www.datadoghq.com/blog/monitor-hashicorp-vault-with-datadog/)",
              "font_size": "14",
              "has_padding": true,
              "show_tick": false,
              "text_align": "left",
              "tick_edge": "right",
              "tick_pos": "50%",
              "type": "note",
              "vertical_align": "center"
          },
```

I believe we want this to be white background as introduction and the blog blurbs in gray